test/unit/org/apache/cassandra/config/YamlConfigurationLoaderTest.java

```diff
@@ -169,6 +171,21 @@ public void readThresholdsFromConfig()
         assertThat(c.row_index_read_size_fail_threshold).isEqualTo(new DataStorageSpec.LongBytesBound(1 << 13, KIBIBYTES));
     }
 
+    @Test
+    public void readSSTableCompressionFromConfig()
+    {
+        Config c = load("test/conf/cassandra.yaml");
+
+        assertNotNull(c.sstable_compressor);
+        assertEquals("org.apache.cassandra.io.compress.LZ4Compressor", c.sstable_compressor.class_name);
```

Comment:
can not use `LZ4Compressor.class.getName()` or similar instead of the explicit string?